src/components/Feed/ProposalFooter.js

```diff
@@ -137,8 +159,33 @@ function DecisionFooter({ proposal, onVoteOnDecision }) {
         <div>
           {votesCount} Vote{votesCount === 1 ? '' : 's'}
         </div>
+        <img
+          css={`
+            margin-left: ${1 * GU}px;
+          `}
+          src={warningSvg}
+          ref={popoverOpener}
+          onClick={handleOpenPopover}
+        />
       </div>
+
       <div>Status: {statusLabel}</div>
+      <Popover
+        visible={warningPopoverVisible}
+        opener={popoverOpener.current}
+        onClose={handleOnClosePopover}
+      >
+        <div
+          css={`
+            padding: ${3 * GU}px;
+            ${textStyle('body2')}
+            width:${compactMode ? 'auto' : 48 * GU}px;
```

Comment:
Let's add the border. Not sure if the color is part of the theme
```suggestion
            width:${compactMode ? 'auto' : 48 * GU}px;
            border: 1px solid #F5A623;
            border-radius: 16px;
```